data-protocols/ids/ids-transform-v1/src/main/java/org/eclipse/dataspaceconnector/ids/transform/IdsProtocol.java

```diff
@@ -15,5 +15,5 @@
 package org.eclipse.dataspaceconnector.ids.transform;
 
 public final class IdsProtocol {
-    public static final String INFORMATION_MODEL_VERSION = "4.0.0";
+    public static final String INFORMATION_MODEL_VERSION = "4.2.7";
```

Comment:
Does this need to be a hardcoded string? Can't we just read the value from the project's `gradle.properties` file? I think this also depends on whether we want to support one single Infomodel version or allow the support of multiple versions in different extensions. Have we already agreed on something? Nevertheless, then we should use a properties/config file instead of defining this in the class, don't we?

Reply:
Absolutely agree with you, having it hardcoded might not be the best way here.
Anyway, we decided to keep it super simple and did it this way. When any upcoming requirement needs a more sophisticated solution we can introduce it then.
As far as I understood from past meetings a connector will only support one IDS info model version at a time. If multiple versions need to be supported, then another connector has to be deployed beside supporting the other version.

Reply:
Sounds good!